### ISSUE TITLE
Expectations in Dutch version of ETM

### ISSUE THREAD

**Issue by marliekeverweij** [quintel/etmodel#2608]
In the Dutch version of the ETM there are expectations by certain people displayed next to the sliders. In the transport section a lot has changed recently, so now these expectations do not really match with the sliders. For example in the image below s a growth-percentage per year is given in the expectation, but the truck slider is now a share-percentage.

![screen shot 2017-12-20 at 13 55 36](https://user-images.githubusercontent.com/32833996/34208514-f706bd8c-e58e-11e7-8b17-ac4cf865677e.png)

What should we do? 

1. Adjust the expectation (where can this be done?)
2. Delete the expectation (it is written more than six years ago)
3. .....

**Comment by michieldenhaan** [comment]
I'm in favour of removing it all together for all sectors. 
Notifying @jorisberkhout, @AlexanderWirtz , @MartLubben in case they have an opinion

**Comment by AlexanderWirtz** [comment]
R E M O V E ! !

**Comment by michieldenhaan** [comment]
![giphy-downsized](https://user-images.githubusercontent.com/32056448/34260408-2371e4aa-e666-11e7-851c-4abfe39b91f1.gif)

**Comment by marliekeverweij** [comment]
Okay I will remove them this week!

**Comment by marliekeverweij** [comment]
One more question. Do we want to be able to use the prediction-functionality in the future or can I **delete all files** related to these predictions?

**Comment by ChaelKruip** [comment]
> One more question. Do we want to be able to use the prediction-functionality in the future or can I delete all files related to these predictions?

Delete them. The beautiful thing about Github is that nothing is really lost ever (at least theoretically)!

**Comment by antw** [comment]
> The beautiful thing about Github is that nothing is really lost ever (at least theoretically)!

Note that this is of course only true for things which are kept in the repository (on GitHub). Information stored in the database – such as the predictions and other data they use – cannot be restored with a simple `git revert`.

We have daily backups of course, but these are not retained forever.

**Comment by antw** [comment]
> It is not working properly yet.

@marliekeverweij What's not working currently? I can see the missing slider labels (I can fix that) and I need to remove some database tables, but a quick glance suggests that everything is otherwise working fine. Is there anything else I might need to fix before merging this?

**Comment by marliekeverweij** [comment]
You are right! I am indeed only missing all slider labels. Furthermore, it is working :)

**Comment by antw** [comment]
Since we both tested it, I merged this without bothering with a pull request. I also squashed it down to a single commit since one commit per deleted file was a bit much 😄. @marliekeverweij: Thanks for taking care of this!
